gc/verbose/VerboseWriterFileLoggingSynchronous.cpp

```diff
@@ -95,7 +95,10 @@ MM_VerboseWriterFileLoggingSynchronous::openFile(MM_EnvironmentBase *env, bool p
 		return false;
 	}
 	
-	_logFileDescriptor = omrfile_open(filenameToOpen, EsOpenRead | EsOpenWrite | EsOpenCreate | EsOpenTruncate, 0666);
+	int32_t openFlags =  EsOpenRead | EsOpenWrite | EsOpenCreate;
+	openFlags |= _manager->fileOpenMode(env);
```

Comment:
why don't we merge it in one line?
```suggestion
	int32_t openFlags =  EsOpenRead | EsOpenWrite | EsOpenCreate | _manager->fileOpenMode(env);
```

Reply:
Thanks! I've made the change here and in `VerboseWriterFileLoggingBuffered.cpp`